app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -105,18 +108,28 @@ private fun HomeSectionHeaderContent(
         )
 
         onShowAllClick?.let {
-            ClickableText(
-                text = AnnotatedString(text = stringResource(id = R.string.recent_tabs_show_all)),
-                modifier = Modifier.padding(start = 16.dp)
-                    .semantics {
-                        contentDescription = description
-                    },
-                style = TextStyle(
-                    color = showAllTextColor,
-                    fontSize = 14.sp,
-                ),
-                onClick = { onShowAllClick() },
-            )
+            Box(
+                Modifier
+                    .defaultMinSize(
+                        minWidth = ButtonDefaults.MinWidth,
+                        minHeight = ButtonDefaults.MinHeight,
+                    )
+                    .clickable(onClick = onShowAllClick),
+                contentAlignment = Alignment.CenterEnd,
+            ) {
+                BasicText(
+                    text = stringResource(id = R.string.recent_tabs_show_all),
+                    modifier = Modifier
+                        .semantics {
+                            contentDescription = description
+                        }
+                        .padding(start = 16.dp),
+                    style = TextStyle(
+                        color = showAllTextColor,
+                        fontSize = 14.sp,
+                    ),
+                )
+            }
```

Comment:
This code looks like a `TextButton` to me now. https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/compose/button/TextButton.kt#28

I don't have an immediate chance to think about this more thoroughly so I will cc @MozillaNoah

Reply:
I agree that this should just be replaced with a `TextButton`. Depending how the Talkback works, I would either add the semantics to the `TextButton` modifier or add a parameter to our composable to pass it down to the inner `Text()` composable.

Reply:
@MozillaNoah Thanks for taking a look!